src/views/UserDetails.vue

```diff
@@ -494,12 +503,15 @@ export default defineComponent({
       imageUrl: "",
       isUserFetched: false,
       showPassword: false,
-      shopifyShopsForProductStore: [] as any
+      shopifyShopsForProductStore: [] as any,
+      isValid: false
```

Comment:
No need to have another variable just to check whether we got the data or not, we can directly check the selected user in the template.